Samenvatting/Afleidingen.tex

```diff
@@ -1204,7 +1204,7 @@ \section{Afleiding 30}
 	\begin{subfigure}{.5\textwidth}
 		\centering
 		\includegraphics[width=0.9\linewidth]{BuigingKlein}
-		\caption[Buiging kleine doorlating]{Buiging door klein gat}
+		\caption[Buiging kleine doorlating // zie video experiment van young minuut 7 , als a << landa dan is er geen propagatie door de opening, dus ook geeen tekening? im not sure mss wilt die tekening juist da zeggen]{Buiging door klein gat}
```

Comment:
Het is inderdaad zo dat als a << \lambda, er geen licht door de opening zal gaan. Hiervoor is er inderdaad geen tekening. Deze tekeningen tonen het geval waarbij dat a>>\lambda (hier gaat het licht door en is er geen buiging) en het geval waarbij a \approx \lambda (hier gaat het licht door maar is er wel buiging).